src/components/FacebookAuthenticationButton/index.ts

```diff
@@ -0,0 +1,53 @@
+import { Stream, constant, just, startWith } from 'most';
+import { VNode, button } from '@motorcycle/dom';
+import { REDIRECT, GET_REDIRECT_RESULT, AuthenticationType }
+  from '../../drivers/firebase-authentication';
+import {
```

Comment:
Why are these imported? Shouldn’t they live in this file? (Sorry, Tylor. I didn’t make these rules. They’ll be killing us.)